person.rb

```diff
@@ -0,0 +1,37 @@
+class Person
+  def initialize(name, age, _parent_permission = 'true')
+    @id = id
+    @name = name
+    @age = age
+  end
+
+  def gets_id
+    @id
+  end
+
+  def gets_name
+    @name
+  end
+
+  def gets_age
+    @age
+  end
+
+  def sets_name(name)
+    @name = name
+  end
+
+  def sets_age(age)
+    @age = age
+  end
```

Comment:
It's better to use attribute accessors instead of creating methods for each.
Example:
```
  attr_accessor :name, :age 
  attr_reader :id 
```

>:id should only be able to be retrieved so only the Getter is defined